Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: docker
```

Comment:
Maybe you could call this `build` (and `registry_docker` `build_registry_docker`). Then also add a `run` that runs the docker image.

Reply:
But that's not good make terminology. You make a target, so a target should be a noun, not a verb. I think I'll change it to `registry_docker_image` though. As for running a DDM based on the images, that will take a bit more work because we need a way to load assets and submit requests and such. That's for some future PR.

Reply:
OK, I guess I have always abused `make` for my evil data science purposes :)

What I mean with `run` is to run this docker image to serve the registry, that should work right?

Reply:
Yeah, that could work, but the question is what you would use a bare registry for. I imagine there being some kind of driver script that starts up a registry and some sites and then controls the whole thing to run an experiment, but I'm not sure how that would work yet, and I think it's better to do in Python than using Make. So I'm thinking to just stick with building, and leave running for the future.